spigot/src/main/java/com/github/syldium/nethertree/commands/ReloadCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.github.syldium.nethertree.commands;
+
+import com.github.syldium.nethertree.NetherTreePlugin;
+import com.github.syldium.nethertree.runnable.RunnablesManager;
+import com.github.syldium.nethertree.util.NetherTreePermissions;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+public class ReloadCommand implements CommandExecutor {
+
+    private final NetherTreePlugin plugin;
+
+    public ReloadCommand(NetherTreePlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String s, @NotNull String[] args) {
+        if (!sender.hasPermission(NetherTreePermissions.NETHER_TREE_ADMIN)) {
+            sender.sendMessage(ChatColor.RED + "You don't have permission to use this command.");
+            return true;
+        }
+        plugin.reloadConfig();
+        if (plugin.getRunnablesManager() != null) {
```

Comment:
The `RunnableManager` should never be null: it's a registry of runnables that may be empty but not null.